Remotion/Web/Development.WebTesting.IntegrationTests/IntegrationTest.cs

```diff
@@ -52,22 +52,29 @@ protected IDriver Driver
     public void IntegrationTestOneTimeSetUp ()
```

Comment:
Not sure if this is done as intended. 

Before, the OneTimeSetup decided based on the fixture declaration. Now, we must decide based on the individual test declaration. 

For now, the idea is that we run each test assembly once, and only run those tests in head-mode that do not support headless. In the future, we can run the test assemblies twice. Once with headless and once with head, each executing only their own subset of tests.

The check as it is implemented now does not work with the require-head attribute applied to tests. The onetime-setup is not executed per test, only per testfixture.

With IntegrationTestSetup, this is different, here we can do it per test. The problem is, that the webtesthelper is already configured for headless. We may need to take a step back and updated the design concept in this type.

Reply:
As for how I understood the ticket: We want to ignore the tests marked with `[RequiresUserInterface]` if we are in headless mode.
But we also want functionality to make the tests in a non-headless mode so we can only run the test which do require it?

Reply:
I believe this this is meant to describe a future desired state. The current step is only meant to make sure only indivdual tests that require a UI are run with head. Before, we could only switch this based on the TestFixture-level but not on the level of individual tests.

The thing is, our build right now runs with headless-by-default and individual testfixtures opt out. if we implement this task as ignoring tests, we would ignore all tests that require head in the build without complensating measures. Therefor, this cannot be the intent :)